### ISSUE TITLE
PDF / Notebook Distribution

### ISSUE THREAD

**Issue by nela** [PacktPublishing/Deep-Reinforcement-Learning-with-Python#2]
Hello,

Came accross this book while researching DRL for my master thesis. Is the book available for purchase electronically? Either as a PDF or a Jupyter Notebook?

Kind Regards

**Comment by sudharsan13296** [comment]
Yes, you can get the book on Amazon - https://www.amazon.com/gp/product/B08HSHV72N/ref=dbs_a_def_rwt_hsch_vapi_tkin_p1_i2

**Comment by sudharsan13296** [comment]
Yes, you can get the book on Amazon - https://www.amazon.com/gp/product/B08HSHV72N/ref=dbs_a_def_rwt_hsch_vapi_tkin_p1_i2
